src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -1177,4 +1218,33 @@ protected static <V extends Enum<V> & Quantity, E extends Enum<E> & Quantity> do
     protected static <V extends Enum<V> & Quantity, E extends Enum<E> & Quantity> double unscaleFunction(LfSensitivityFactor<V, E> factor, double value) {
         return value * getFunctionBaseValue(factor);
     }
+
+    protected static int getLegNumber(SensitivityFunctionType type) {
```

Comment:
It would make sense to put that int (or Integer) inside the enum, don't you think?